tests/ctst/world/Zenko.ts

```diff
@@ -58,38 +137,40 @@ export default class Zenko extends World {
      * @constructor
      * @param {Object} options - parameters provided as a CLI parameter when running the tests
      */
-    constructor(options: any) {
+    constructor(options: IWorldOptions) {
         super(options);
 
         // store service users credentials from world parameters
-        if (this.parameters.ServiceUsersCredentials) {
-             Object.entries(JSON.parse(this.parameters.ServiceUsersCredentials))
-                .forEach((entry : [string, any]) => {
-                    Zenko.serviceUsersCredentials[entry[0] as string] = {
-                        AccessKeyId: entry[1].accessKey as string,
-                        SecretAccessKey: entry[1].secretKey as string,
-
+        if ((this.parameters as Parameters).ServiceUsersCredentials) {
```

Comment:
Why not setting the `Parameters` type directly in `parameters`?

Reply:
Unfortuneatly because parameters is an `IWorld` attribute from cucumber that is `any`; is there a better way given this?